pandas/core/arrays/datetimelike.py

```diff
@@ -211,6 +219,10 @@ def astype(self, dtype, copy=True):
     # ------------------------------------------------------------------
     # Null Handling
 
+    def isna(self):
+        # EA Interface
+        return self._isnan
```

Comment:
Is it needed to have the `_isnan` concept on the arrays? We use it in some internal methods on the Index class, but for Arrays it seems to me additional complexity compared to simply defining `isna` appropriately on each Array ?

Reply:
Discussed elsewhere; can we mark as resolved?